### DISCUSSION TITLE
Component that wraps ConfirmDialog

### DISCUSSION THREAD

**Discussion by Phil-G** [primefaces/community#272] [PrimeNG]
I tried creating a component that wraps ConfirmDialog so I can customize the `<p-confirmDialog>` templates (header, footer, etc.) for re-use on several pages.
`this.confirmationService.confirm()` doesn't seem to be able to find the `<p-confirmDialog>` tags when they are in the component HTML but they are found if I take them out of the component HTML and embed them in the page directly. Is there a limitation on how PrimeNG locates tags? I am using the `key` property.

[Stackblitz ConfirmDialog](https://stackblitz.com/edit/gcdrhb?file=src%2Fapp%2Fdemo%2Fconfirm-dialog-basic-demo.html)

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue precisely?

**Reply by Phil-G** [reply]
I modified the PrimeNG demo.
[Stackblitz ConfirmDialog](https://stackblitz.com/edit/gcdrhb?file=src%2Fapp%2Fdemo%2Fconfirm-dialog-basic-demo.html)

**Reply by cetincakiroglu** [reply]
Unfortunately, there isn't an input property to achieve this. You can open an issue and request this feature or you can ask our community PrimeLand discord channel https://discord.gg/gzKFYnpmCY
